lib/diagnostic.js

```diff
@@ -17,7 +17,19 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+Description: Calender program for keeping track of things
+Entity: Event
+Attributes: when, where, duration
+Methods: conflictingEvent
+
+Entity: User
+Attributes: name, address, e-mail
+Methods: schedule, outOfTown
```

Comment:
`schedule` is also a little ambiguously named. Do you mean "schedule a new event"? "Show the current schedule"?